crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -90,6 +90,18 @@ def addons
       end
     end
 
+    def repocheck
+      # FIXME: once we start working on 7 to 8 upgrade we have to adapt the sles version
+      {
+        cloud: {
+          available: repo_version_available("suse-openstack-cloud", "8") == 0
+        },
+        sles12sp2: {
```

Comment:
As this is not the value we are checking for, maybe it could be more generic, like "system" (as opposed to "cloud")